scripts/world_bank/datasets/datasets.py

```diff
@@ -453,27 +516,34 @@ def write_observations_from_zip(zip_file, svs):
 
 
 def get_observations_from_zip(zip_file, svs):
-    with zipfile.ZipFile(zip_file, 'r') as zip:
-        (data_file, _) = get_data_and_series_file_names(zip)
-        if data_file is None:
-            logging.warning('No data file found in ZIP file: %s', zip_file)
-            return []
-        else:
-            # Use name of file (excluding the extension) as the measurement method
-            measurement_method = f"{WORLD_BANK_MEASUREMENT_METHOD_PREFIX}_{zip_file.split('/')[-1].split('.')[0]}"
-            with zip.open(data_file, 'r') as csv_file:
-                data_rows = sanitize_csv_rows(
-                    list(csv.DictReader(codecs.iterdecode(csv_file, 'utf-8'))))
-                num_rows = len(data_rows)
-                logging.info('# data rows in %s: %s', zip_file, num_rows)
-
-                obs_csv_rows = []
-                for data_row in data_rows:
-                    obs_csv_rows.extend(
-                        get_observations_from_data_row(data_row, svs,
-                                                       measurement_method))
-
-                return obs_csv_rows
+    try:
+        with zipfile.ZipFile(zip_file, 'r') as zip:
+            (data_file, _) = get_data_and_series_file_names(zip)
+            if data_file is None:
+                logging.warning('No data file found in ZIP file: %s', zip_file)
+                return []
+            else:
+                # Use name of file (excluding the extension) as the measurement method
+                measurement_method = f"{WORLD_BANK_MEASUREMENT_METHOD_PREFIX}_{zip_file.split('/')[-1].split('.')[0]}"
+                with zip.open(data_file, 'r') as csv_file:
+                    data_rows = sanitize_csv_rows(
+                        list(
+                            csv.DictReader(codecs.iterdecode(csv_file,
+                                                             'utf-8'))))
+                    num_rows = len(data_rows)
+                    logging.info('# data rows in %s: %s', zip_file, num_rows)
+
+                    obs_csv_rows = []
+                    for data_row in data_rows:
+                        obs_csv_rows.extend(
+                            get_observations_from_data_row(
+                                data_row, svs, measurement_method))
+
+                    return obs_csv_rows
+    # Exception can be ignored as there might be some corrupted zip files from source
+    except Exception as e:
+        logging.info("There is problem while processing the zip file: %s", e)
```

Comment:
may help to add the file name too:
`logging.info("Error  processing the zip file %s: %s", zip_file, e)`